src/screens/CategoriesFilterScreen/component.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import React, { PureComponent } from "react";
+import { View, StyleSheet, TouchableOpacity } from "react-native";
+import SafeAreaView from "react-native-safe-area-view";
+import type { NavigationScreenProp, NavigationState } from "react-navigation";
+import type { Event } from "../../data/event";
+import Text from "../../components/Text";
+import {
+  eucalyptusGreenColor,
+  lightNavyBlueColor
+} from "../../constants/colors";
+import text from "../../constants/text";
+import ContentPadding from "../../components/ContentPadding";
+import Header from "./Header";
+import List from "./List";
+import locale from "../../data/locale";
+
+type Props = {
+  navigation: NavigationScreenProp<NavigationState>,
+  events: Event[],
+  stagedCategories: Set<string>,
+  toggleCategoryFilter: Function,
+  onApplyFilters: Function,
+  onClearAll: Function,
+  onClose: Function
```

Comment:
If you know the signature of the function, you can tell flow: `onClose: () => {}`